QUEST_JOBSTEP.tsv

```diff
@@ -2135,53 +2135,53 @@ QUEST_JOBSTEP_20150317_002134	Go meet Doppelsoeldner Master in Forgotten Port.
+QUEST_JOBSTEP_20150317_002158	Meet the Psychokino Master in Siauliai Miners' Village
 QUEST_JOBSTEP_20150317_002159	Defeat the Werewolf in Siauliai Miners' Village
-QUEST_JOBSTEP_20150317_002160	Psychokino Master is busy but doen't want to pass on the chance of defeating the Werewolf to Linker Master. Defeat the Werewolf in Miners' Village.
-QUEST_JOBSTEP_20150317_002161	Defeated the Werewolf. Go back and report to Psychokino Master.
-QUEST_JOBSTEP_20150317_002162	Meet Linker Master
-QUEST_JOBSTEP_20150317_002163	Meet Linker Master in Siauliai Miners' Village
-QUEST_JOBSTEP_20150317_002164	Get rid of the Werewolf in Siaulai Miner's Village
-QUEST_JOBSTEP_20150317_002165	Linker Master wants a Linker apprentice to defeat the Werewolf and show Psychokino Master who's the boss. Defeat the Werewolf in Siauliai Miners' Village.
-QUEST_JOBSTEP_20150317_002166	Defeated the Werewolf. Go back and report to Linker Master.
-QUEST_JOBSTEP_20150317_002167	Meet Thaumaturge Master
-QUEST_JOBSTEP_20150317_002168	Meet Thaumaturge Master in Fedimian
-QUEST_JOBSTEP_20150317_002169	Defeat Nederbovines in Fault Line.
-QUEST_JOBSTEP_20150317_002170	Thaumaturge Master told you that all the Masters are working toghther to defeat the monsters that are making mess so if you want to be a Thaumaturge, you should participate as well. Defeat Nederbovines that are making mess in Fault Line.
-QUEST_JOBSTEP_20150317_002171	Defeated Netherbovine. Go back and report to Thaumaturge Master.
-QUEST_JOBSTEP_20150317_002172	Meet Elementalist Master
-QUEST_JOBSTEP_20150317_002173	Meet Elementalist Master in Fedimian
+QUEST_JOBSTEP_20150317_002160	The Psychokino Master is busy but doesn't want to pass on the chance of defeating the Werewolf before the Linker Master can do it. Defeat the Werewolf in Miners' Village.
+QUEST_JOBSTEP_20150317_002161	You defeated the Werewolf. Report back to the Psychokino Master.
+QUEST_JOBSTEP_20150317_002162	Meet the Linker Master
+QUEST_JOBSTEP_20150317_002163	Meet the Linker Master in Siauliai Miners' Village
+QUEST_JOBSTEP_20150317_002164	Defeat the Werewolf in Siaulai Miner's Village
+QUEST_JOBSTEP_20150317_002165	The Linker Master wants a Linker apprentice to defeat the Werewolf and show the Psychokino Master who's the boss. Defeat the Werewolf in Siauliai Miners' Village.
+QUEST_JOBSTEP_20150317_002166	You defeated the Werewolf. Report back to the Linker Master.
+QUEST_JOBSTEP_20150317_002167	Meet the Thaumaturge Master
+QUEST_JOBSTEP_20150317_002168	Meet the Thaumaturge Master in Fedimian
+QUEST_JOBSTEP_20150317_002169	Defeat Nederbovines in Fault Line
+QUEST_JOBSTEP_20150317_002170	The Thaumaturge Master told you that all the Masters are working together to defeat the monsters that are causing trouble in Fault Line so if you want to become Thaumaturge, you should participate as well. Defeat Nederbovines that are messing around in Fault Line.
+QUEST_JOBSTEP_20150317_002171	You defeated enough Netherbovine. Report back to the Thaumaturge Master.
```

Comment:
You defeated enough Netherbovine -> You defeated Nederbovine. 
(*If you write down Nederbovine before, do it in this line as well for consistency.*)